packages/components/data-table-manager/src/data-table-settings/data-table-settings.tsx

```diff
@@ -34,6 +34,7 @@ type MappedColumns = Record<string, TColumnData>;
 export type TDropdownOption = {
   value: string;
   label: string;
+  isDisabled?: boolean;
```

Comment:
Not sure if this shows up in the README.md, but maybe just run `yarn generate-readmes` to figure out if something changed and commit if necessary.